<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>671</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btn_msg_01">
          <property name="text">
           <string>Camera Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_send_image">
          <property name="text">
           <string>Send Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_msg_02">
          <property name="text">
           <string>Send Check 2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QWidget" name="ImgWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="ImgReturn" native="true">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="pte_console">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="openGLWidget"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
